docs/eso/eso.rst

```diff
@@ -51,14 +49,14 @@ interaction with the ESO archive.
     Authenticating TEST on www.eso.org...
     Authentication failed!
     >>> # Second example: pretend ICONDOR is a valid username
-    >>> eso.login("ICONDOR", store_password=True) # doctest: +SKIP
-    ICONDOR, enter your ESO password:
+    >>> eso.login("Tinuade", store_password=True)  # doctest: +SKIP
```

Comment:
I would say remove these changes and use the more generic username we had before, as well as the skip here as the whole section is skipped.

Reply:
Did the more generic username, but it turns out that because of the blank line in the output the `doctest: +SKIP` is necessary for the second two calls.